src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/linalg/FourierLinearOperatorNDArrayUnitTest.java

```diff
@@ -0,0 +1,128 @@
+package org.broadinstitute.hellbender.tools.coveragemodel.linalg;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.junit.Assert;
+import org.nd4j.linalg.factory.Nd4j;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Unit tests for {@link FourierLinearOperatorNDArray}
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+
+public class FourierLinearOperatorNDArrayUnitTest extends BaseTest {
```

Comment:
A few exact cases would be helpful here, both for making the tests more compelling but also for clarifying the API of the tested class.  For example, construct sinusoidal test data and explicitly check that it is an eigenvector.